content/en/blog/fsm3/index.md

```diff
@@ -0,0 +1,355 @@
+---
+date: 2024-02-06
+title: "Issuing Service"
+linkTitle: "Issuing Service"
+description: "
+Our SSI/DID agency solution is fully symmetric, which means that our agents can
+be in any role simultaneously: `issuer`, `holder`, and `verifier`. However, the
+base-technology (`indylib`) haven't been built for symmetry. Luckily we have our
+own protocol engine and the tools around it allow us to build SSI services as
+chatbots. We have implemented our own state machine engine to develop and run
+these chatbot machines.
+"
+author: Harri Lainio
+resources:
+- src: "**.{png,jpg}**"
+  title: "Image #:counter"
+---
+
+This post is a development story (or hands-on workshop) of how we can use an FSM
+chatbot to implement SSI Services to allow any SSI/DID owner to be an issuer.
+Let’s start with the result and see what the state machine looks like. As you
+can see below, it’s simple, elegant, and, most importantly, easy to reason.
+
+```plantuml @startuml
```

Comment:
The diagram overflows from the view. How about scaling the size with plantuml params (and a link to full-size version), or replacing it with an image?